aws/requests.yaml

```diff
@@ -0,0 +1,6 @@
+releases:
+    - name: "> 11.5.0, < 12.0.0"
```

Comment:
Why not make this >12.0.0?
From what I understood here: https://github.com/giantswarm/giantswarm/issues/12334#issuecomment-664179537
The PM is not impacted by this version anymore as AWS CNI deals with it. We should still bump calico but it does not really need to be higher than 3.10.X in the 11.X.X releases IMO.

Reply:
I thought it would be preferable to simply align the versions, but I can add a separate rule for 12+ and keep 11.x at 3.10

Reply:
Yes - I would prefer to align on 12+. I think that's reasonable and should still cover us very soon as all teams have / are working on 12.X releases.

Reply:
Ok, AWS 11.5.0 already includes 3.10.4, so I'll remove the request for anything on 11.x. 
The AWS 12.0.0 PR includes calico 3.15.1, so I'll request baselining the next 12.x for KVM and Azure on that as well. Then 12.x will cover everything we currently have open for calico.